### PR TITLE
[Fix](bangc-ops): fix roipointpool3d interface description error.

### PR DESCRIPTION
**Pull request by DanieeelLiu**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

mluOpRoipointPool3dForward，mluOpActiveRotatedFilterForward, mluOpThreeNNforward, mluOpPsamaskForward, mluOpPsamaskBackward, mluOpBoxIouRotated, mluOpDiffIouRotatedSortVerticesForward 接口返回值描述修改，其他算子缺少  MLUOP_STATUS_EXECUTION_FAILED 的进行统一添加

## 2. Modification

bangc-ops/mlu_op.h

### CODE REVIEW

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -3863,7 +3865,7 @@ mluOpDynamicPointToVoxelForward(const mluOpHandle_t handle,
  *
  * @par Return
  * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
- *   ::MLUOP_STATUS_NOT_SUPPORTED
+ *   ::MLUOP_STATUS_NOT_SUPPORTED,
```
Comment (PetrelYy) [inline bangc-ops/mlu_op.h:3868]: 多了个逗号

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -11174,7 +11192,8 @@ mluOpGetReduceOpWorkspaceSize(mluOpHandle_t handle,
  * ::mluOpActiveRotatedFilterForward.
  *
  * @par Return
- * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM, ::MLUOP_STATUS_NOT_SUPPORTED,
+ *   ::MLUOP_STATUS_EXECUTION_FAILED
```
Comment (PetrelYy) [inline bangc-ops/mlu_op.h:11196]: mluOpGetActiveRotatedFilterForwardWorkspaceSize 没这个返回值

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -11447,7 +11467,8 @@ mluOpDeformRoiPoolForward(const mluOpHandle_t handle,
  * Pointer to the MLU memory that stores the gradient of the offset tensor.
  *
```
Comment (PetrelYy) [inline bangc-ops/mlu_op.h:11468]: 11470 前缺了 mluOpGetIndicePairs，mluOpActiveRotatedFilterForward，mluOpDeformRoiPoolForward

File: bangc-ops/mlu_op.h [hunk 4]
```diff
@@ -12741,7 +12766,8 @@ mluOpIndiceConvolutionForward(mluOpHandle_t handle,
  * Pointer to the MLU memory that stores the \b dispatch tensor.
  *
  * @par Return
```
Comment (PetrelYy) [inline bangc-ops/mlu_op.h:12768]: 缺了 mluOpIndiceConvolutionForward

**Review by mahxn0 — APPROVED** [review]

File: bangc-ops/mlu_op.h (hunk 3 above)
Comment (DanieeelLiu) [reply bangc-ops/mlu_op.h:11467]: mluOpGetIndicePairs 调用二进制算子，没有使用kernel_check 宏， mluOpDeformRoiPoolForward 本身完备， mluOpActiveRotatedFilterForward 已重新修改

File: bangc-ops/mlu_op.h [hunk 5]
```diff
@@ -12329,7 +12352,8 @@ mluOpGetRoiPointPool3dWorkspaceSize(mluOpHandle_t handle,
  * @param[out] pooled_empty_flag
  * Pointer to the MLU memory that stores the second output tensor.
  * @par Return
- * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM, ::MLUOP_STATUS_NOT_SUPPORTED
```
[the remark this answers is not in the document]
Comment (DanieeelLiu) [reply bangc-ops/mlu_op.h:12332]: 同 mluOpGetIndicePairs

File: bangc-ops/mlu_op.h (hunk 4 above)
Comment (DanieeelLiu) [reply bangc-ops/mlu_op.h:12767]: 同 mluOpGetIndicePairs

File: bangc-ops/mlu_op.h (hunk 1 above)
Comment (DanieeelLiu) [reply bangc-ops/mlu_op.h]: Fixed

File: bangc-ops/mlu_op.h (hunk 2 above)
Comment (DanieeelLiu) [reply bangc-ops/mlu_op.h]: Fixed

**Review by PetrelYy — APPROVED** [review]

**Review by Unireverse — APPROVED** [review]

**Pull request merged**